### PR TITLE
Add silent mode

### PR DESCRIPTION
**Pull request by donno2048**

### CODE REVIEW

**Comment by donno2048** [comment]
All that left is to document the change

File: ref/__init__.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-__protectfiles = None
+__protectfiles, __silent = None, None
 __restrict = {
     "os": ["system", "popen", "kill", "spawn", "execl", "execle", "execlp", "execlpe", "execv", "execve", "execvp", "execvpe", "killpg", "fork", "forkpty", "plock"],
     "subprocess": ["run", "check_output", "call"],
     "pathlib.Path": [],
     "shutil": []
 }
-def main(__builtins__: ModuleType, protectfiles: bool = False, protectdirs: bool = False, lockperms: bool = False) -> None:
+def main(__builtins__: ModuleType, protectfiles: bool = False, protectdirs: bool = False, lockperms: bool = False, silent: bool = False) -> None:
```
Comment (iam-py-test) [inline ref/__init__.py:11]: This looks great. 
Let me test...

**Review by iam-py-test — APPROVED** [review]

File: ref/__init__.py (hunk 1 above)
Comment (iam-py-test) [reply ref/__init__.py:11]: Tested; everything works perfectly well

**Comment by iam-py-test** [comment]
@donno2048 I was able to run __init__.main with silent mode enabled _after_ running it the first time with silent mode disabled, and it switched to silent mode.
Is this by design or is this a bug?
Should I open another issue

**Comment by iam-py-test** [comment]
I need to get on a Zoom call so won't see notifications.
Will check after the call...

**Comment by donno2048** [comment]
> @donno2048 I was able to run **init**.main with silent mode enabled _after_ running it the first time with silent mode disabled, and it switched to silent mode.
> Is this by design or is this a bug?
> Should I open another issue

It's simply because the import function is being changed, it's perfectly fine

**Pull request merged**